src/cljs/main/broadfcui/page/external_importer.cljs

```diff
@@ -0,0 +1,68 @@
+(ns broadfcui.page.external-importer
+  (:require
+   [dmohs.react :as react]
+   [broadfcui.common.links :as links]
+   [broadfcui.common.style :as style]
+   [broadfcui.config :as config]
+   [broadfcui.nav :as nav]
+   [broadfcui.utils :as utils]
+   ))
+
+(defn render-logged-out-import-tutorial [sign-in-handler]
+  [:div {}
+   [:h1 {:style {:marginBottom "0.3rem"}} "Import Docker Image to FireCloud"]
+   "In order to import data into FireCloud, you'll need two things:"
+   [:div {:style {:marginTop "1.5rem"}}
+    [:div {:style {:width "50%" :display "inline-block"}}
+     [:div {:style {:display "inline-block"
+                    :background-color (:button-primary style/colors) :color "white"
+                    :fontSize "1.25rem" :textAlign "center" :lineHeight "2rem"
+                    :height "2rem" :width "2rem" :borderRadius "50%"}}
+      "1"]
+     [:div {:style {:display "inline-block" :verticalAlign "top" :marginLeft "0.5rem"}}
+      [:h2 {:style {:margin "0 0 0.3rem"}} "Google Billing Project"]
```

Comment:
Do we need the explicit top/side margins of 0 to override h2 defaults?

Reply:
Just the top; the sides are there so that I don't have separate props. Would you prefer that?